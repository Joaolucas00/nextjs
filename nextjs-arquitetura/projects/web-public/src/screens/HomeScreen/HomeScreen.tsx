import { Text } from "@alura/design-system/components/Text";
import { sum } from "@alura/utils/math/sum";

export default function HomeScreen() {
  return (
    <main>
      <h1>Home</h1>
      <Text tag="p">
        Importando modulo local: @alura/utils/math/sum sum(2, 2): {sum(2, 2)}{" "}
      </Text>
      <Text tag="h1">design-system</Text>
    </main>
  );
}
