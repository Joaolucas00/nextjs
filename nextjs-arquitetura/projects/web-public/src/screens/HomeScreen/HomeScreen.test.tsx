import HomeScreen from "./HomeScreen";
import { customRender } from "@alura/test-commons/react-testing-library";

const render = customRender();

describe("HomeScreen", () => {
  test("Renderiza a pÃ¡gina", () => {
    const { container } = render(<HomeScreen />);

    expect(container).toMatchSnapshot();
  });
});
