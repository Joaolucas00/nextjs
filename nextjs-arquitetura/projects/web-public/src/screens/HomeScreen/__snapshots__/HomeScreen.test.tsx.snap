// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen Renderiza a p√°gina 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <main>
    <h1>
      Home
    </h1>
    <p
      class="c0"
    >
      Importando modulo local: @alura/utils/math/sum sum(2, 2): 
      4
       
    </p>
    <h1
      class="c0"
    >
      design-system
    </h1>
  </main>
</div>
`;
